src/core/ext/filters/client_channel/parse_address.h

```diff
@@ -38,6 +38,7 @@
 
 #include "src/core/ext/filters/client_channel/uri_parser.h"
 #include "src/core/lib/iomgr/resolve_address.h"
+#include "src/core/lib/iomgr/sockaddr.h"
```

Comment:
I'd prefer to stick with the include.  I don't want to get into the business of trying to forward-declare structs that are provided by the system header files.